src/core/Akka.API.Tests/CoreAPISpec.ApproveCluster.approved.txt

```diff
@@ -186,6 +186,7 @@ namespace Akka.Cluster
         public System.TimeSpan SeedNodeTimeout { get; }
         public System.TimeSpan UnreachableNodesReaperInterval { get; }
         public string UseDispatcher { get; }
+        public bool VerboseGossipReceivedLogging { get; }
```

Comment:
Added a new setting to prevent this from flooding the logs when `DEBUG` is turned on.